### ISSUE TITLE
[App Request] Joplin 

### ISSUE THREAD

**Issue by Rixiu** [IceWhaleTech/CasaOS-AppStore#58]

**Comment by vjanomolee** [comment]
Yes PLEASE!  I am having a bear of a time trying to set up a Joplin sync server on my Zimaboard with CasaOS. Would be great if it was just an app in the CasaOS store
